src/f64/math.rs

```diff
@@ -1,4 +1,4 @@
-#[cfg(feature = "libm")]
+#[cfg(any(feature = "libm", feature = "nostd-libm"))]
```

Comment:
Should be `#[cfg(any(feature = "libm", all(feature = "nostd-libm", not(feature = "std"))))]`?